templates/web/sutton/waste/header.html

```diff
@@ -0,0 +1,44 @@
+[%
```

Comment:
I'd rather not override the whole template unless absolutely necessary. In this case, can you not put the extra `sutton-gradient` in templates/web/sutton/header_site.html which is included at the appropriate bit, then this template isn't needed, if I've understood it right?